gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -1885,6 +1885,7 @@ gum_emit_import (const GumImportDetails * details,
 {
   GumEnumerateImportsContext * ctx = user_data;
   GumImportDetails d;
+  Dl_info info;
```

Comment:
It looks like this variable can be moved down to the next scope inside this one.

Reply:
oh yeah right, because the string we use doesn't actually live on the stack